contrib/codegen/templates/input/input_snapshot.gotmpl

```diff
@@ -415,3 +478,22 @@ func (b *singleClusterBuilder) insert{{ $kindPlural }}(ctx context.Context, {{ $
 
 {{- end }}
 {{- end }}
+
+func verifyCrdRegistered(ctx context.Context, cfg *rest.Config, group, version, kind string) (bool, error) {
```

Comment:
Same nit as above, why generate this for every snapshot?

Reply:
no longer relevant, the code has been moved